src/ralph/attachments/forms.py

```diff
@@ -27,6 +30,31 @@ def save(self, commit=True):
             * mime_type - uploaded file's content type.
         """
         obj = super().save(commit=False)
+        md5 = Attachment.get_md5_sum(obj.file)
```

Comment:
please describe what is happening here and what is _parent_object (and how it's inserted here)